### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AniruddhaKanhere**
Update the README.md file with latest instructions/directions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +29,15 @@ The porting guide is available on [this page](http://www.FreeRTOS.org/FreeRTOS-P
 
 ## Repository structure
 This repository contains the FreeRTOS-Plus-TCP repository and a number of supplementary libraries for testing/PR Checks.
+Below is the breakdown of what each directory contains:
+- tools
+    - This directory contains the tools and related files (CMock/uncrustify) required for tests/checks for the TCP source code.
```
Comment (gitwallit) [inline README.md:34]: Minor correction in grammar "This directory contains the tools and related files (CMock/uncrustify) required to run  tests/checks on the TCP source code"

File: README.md [hunk 2]
```diff
@@ -27,6 +29,15 @@ The porting guide is available on [this page](http://www.FreeRTOS.org/FreeRTOS-P
...
+- tests
+    - This directory contains all the tests (unit tests and CBMC) and the dependencies ([FreeRTOS-Kernel](https://github.com/FreeRTOS/FreeRTOS-Kernel)/[Litani-port](https://github.com/awslabs/aws-build-accumulator)) the tests require.
+- portable
+    - This directory contains the portable files required to compile the FreeRTOS-Plus-TCP source code for different hardwares/compilers.
```
Comment (gitwallit) [inline README.md:38]: hardware, instead of hardwares

File: README.md [hunk 3]
```diff
@@ -27,6 +29,15 @@ The porting guide is available on [this page](http://www.FreeRTOS.org/FreeRTOS-P
...
+- include
+    - The include directory has all 'core' the header files of FreeRTOS-Plus-TCP source.
```
Comment (gitwallit) [inline README.md:40]: has all the 'core' header files of the FreeRTOS-PLUS-TCP source

File: README.md (hunk 1 above)
Comment (AniruddhaKanhere) [reply README.md]: Thanks @gitwallit, I have updated the readme file.
Can you please re-review it?

File: README.md (hunk 3 above)
Comment (AniruddhaKanhere) [reply README.md]: Oops! What was I thinking... Have updated the language.

**Review by gitwallit — APPROVED** [review]

**Review by yanjos-dev — APPROVED** [review]

**Pull request merged**